<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>367</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>347</width>
     <height>320</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Sync</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>129</y>
       <width>319</width>
       <height>118</height>
      </rect>
     </property>
     <property name="title">
      <string>Addition</string>
     </property>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>301</width>
        <height>81</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QCheckBox" name="us_phonetic">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>US phonetic</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="uk_phonetic">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>UK phonetic</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QCheckBox" name="phrase">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Phrase</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="phraseExplain">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Phrase explain</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QRadioButton" name="fromWordbook">
           <property name="text">
            <string>From WordBook</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="fromYoudaoDict">
           <property name="text">
            <string>From Youdao.com</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>10</y>
       <width>319</width>
       <height>119</height>
      </rect>
     </property>
     <property name="title">
      <string>Main</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>301</width>
        <height>33</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Deck</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="deck"/>
       </item>
       <item>
        <widget class="QPushButton" name="sync">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Sync</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>75</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Please Input the Deck name you want to sync to</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>260</y>
       <width>321</width>
       <height>20</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QProgressBar" name="progress">
        <property name="value">
         <number>0</number>
        </property>
        <property name="textVisible">
         <bool>false</bool>
        </property>
        <property name="format">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="progressLabel">
        <property name="text">
         <string>Fecting Words</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Login</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>321</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>Login</string>
     </property>
     <widget class="QLineEdit" name="username">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QLineEdit" name="password">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="maxLength">
       <number>32768</number>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>61</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Username:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loginTest">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>226</x>
        <y>26</y>
        <width>91</width>
        <height>63</height>
       </rect>
      </property>
      <property name="text">
       <string>Check</string>
      </property>
     </widget>
     <widget class="QLabel" name="loginStatus">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>301</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Press buttom to check Login validation!</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="cloudAPI">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>321</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>Cloud API</string>
     </property>
     <widget class="QLineEdit" name="appID">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::PasswordEchoOnEdit</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="appKey">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::PasswordEchoOnEdit</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>61</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>App Key:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>App ID:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="apiTest">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>226</x>
        <y>27</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Check</string>
      </property>
     </widget>
     <widget class="QLabel" name="apiStatus">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>301</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Select 'From Youdao' radioButtom first</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.SF NS Text'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;a href=&quot;http://ai.youdao.com/&quot;&gt;&lt;span style=&quot; font-size:18pt; text-decoration: underline; color:#0000ff;&quot;&gt;Visit Youdao API&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="fromPublicAPI">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>181</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Public API</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPlainTextEdit" name="debug">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>341</width>
     <height>261</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="backgroundVisible">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>fromYoudaoDict</sender>
   <signal>toggled(bool)</signal>
   <receiver>phrase</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>234</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fromWordbook</sender>
   <signal>toggled(bool)</signal>
   <receiver>phrase</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fromYoudaoDict</sender>
   <signal>toggled(bool)</signal>
   <receiver>us_phonetic</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>207</y>
    </hint>
    <hint type="destinationlabel">
     <x>98</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fromYoudaoDict</sender>
   <signal>toggled(bool)</signal>
   <receiver>uk_phonetic</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>113</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fromYoudaoDict</sender>
   <signal>toggled(bool)</signal>
   <receiver>cloudAPI</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>207</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fromWordbook</sender>
   <signal>toggled(bool)</signal>
   <receiver>cloudAPI</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fromWordbook</sender>
   <signal>toggled(bool)</signal>
   <receiver>fromPublicAPI</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>106</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fromYoudaoDict</sender>
   <signal>toggled(bool)</signal>
   <receiver>fromPublicAPI</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fromPublicAPI</sender>
   <signal>toggled(bool)</signal>
   <receiver>cloudAPI</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>phrase</sender>
   <signal>toggled(bool)</signal>
   <receiver>phraseExplain</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fromWordbook</sender>
   <signal>toggled(bool)</signal>
   <receiver>phraseExplain</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
